frontend/src/components/Board.jsx

```diff
@@ -1,18 +1,15 @@
-import { useState } from "react";
+import React, { useState } from "react";
 import Cell from "./Cell";
 import Keypad from "./Keypad";
 
 function Board() {
   const [sudokuGrid, setSudokuGrid] = useState(Array.from({ length: 9 }, () => Array(9).fill("")));
-  const [selectedCell, setSelectedCell] = useState({ row: null, col: null });
-
-  const handleCellChange = (row, col, e) => {
-    const value = e.target.value;
-    if (/^[1-9]$/.test(value) || value === "") {
-      const newGrid = [...sudokuGrid];
-      newGrid[row][col] = e.target.value;
-      setSudokuGrid(newGrid);
-    }
```

Comment:
Character input handling was deleted, please add this back in

Reply:
fixed